bvm/ballerina-core/src/main/java/org/ballerinalang/model/values/BXMLSequence.java

```diff
@@ -167,7 +166,7 @@ public void setAttribute(String localName, String namespace, String prefix, Stri
             return ((BXMLItem) sequence.getRefValue(0)).getAttributesMap();
         }
 
-        return new BMap<>(new BMapType(BTypes.typeString));
+        return null;
```

Comment:
Only XML items are allowed to return a map when ask for attribute map, XML sequence should return a nil value.